src/components/Modals/NewCourse/NewCourseModal.vue

```diff
@@ -85,6 +90,9 @@ export default defineComponent({
       this.selectedCourse = result;
       this.getReqsRelatedToCourse(result);
     },
+    escape() {
```

Comment:
I think `closeCurrentModal` would be a more self-documenting name (that is consistent with the other modals) but is there a reason why you opted to call this method `escape`?